src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -581,7 +581,9 @@ protected function getEnvParameters()
     protected function buildContainer()
     {
         foreach (array('cache' => $this->getCacheDir(), 'logs' => $this->getLogDir()) as $name => $dir) {
-            if (!is_dir($dir)) {
+            if (false === $dir && 'logs' === $name) {
```

Comment:
Caching is a feature of the kernel, while logging is a feature of monolog. So I would so no.